<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="ptRxData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leComPortName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM33</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConnect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSend">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTxText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRst">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCwjap">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CWJAP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCwqap">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CWQAP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbVer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VER</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbState">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>271</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>Состояние модуля ESP8266</string>
    </property>
    <widget class="QCheckBox" name="cbModuleReady">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Got ready</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbModuleAtOk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Got AT --&gt; OK</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leStaMac">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Station MAC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leStaMac_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Station IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leParserMode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current parser mode</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>ptRxData</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
